app/Services/Search/Traits/ValidatesTerm.php

```diff
@@ -36,10 +36,10 @@ private function couldBePublicKey(string $term): bool
      */
     private function couldBeUsername(string $term): bool
     {
-        $regex = '/^[a-zA-Z0-9!@$&_.]+$/';
+        $regex = '/^[a-zA-Z0-9!@$&_. ]+$/';
```

Comment:
I take it https://github.com/ArkEcosystem/common/blob/master/mainnet/known-wallets-extended.json#L9 still won't be found as it includes `()`? 🤔